easybuild/easyconfigs/o/OpenMM/OpenMM-6.3-foss-2015a.eb

```diff
@@ -0,0 +1,26 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Author:    Stephane Thiell <sthiell@stanford.edu>
+##
+name = 'OpenMM'
+version = '6.3'
+
+homepage = 'http://openmm.org/'
+description = "A high performance toolkit for molecular simulation."
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+source_urls = ['https://github.com/pandegroup/openmm/releases']
+sources = [SOURCELOWER_TAR_GZ]
+
+builddependencies = [
+    ('CMake', '3.2.3'),
+    ('libxml2', '2.9.2'),
+    ('SWIG', '3.0.8', '-Python-2.7.11'),
+    ('Doxygen', '1.8.7'),
+]
+
+dependencies = [('Python', '2.7.11')]
```

Comment:
@boegel sorry about that! When I installed it on our systems, I tested this easyconfig with different versions of CUDA, and I just loaded the right version in the build environment before launching eb. It's bad practice and I don't do that anymore. It is important to note that CUDA 7.5 is not recommended for OpenMM 6.3 (cf. https://github.com/pandegroup/openmm/pull/1249), so we're using CUDA/7.0.28 in production, but this might not be the case anymore for the latest 6.3.1 release of OpenMM (not using in production yet).